packages/frontend-web/src/app/scenes/Settings/store.ts

```diff
@@ -108,15 +108,15 @@ export function updateTags(oldTags: List<ITagModel>, newTags: List<ITagModel>):
 async function addRule(rule: IRuleModel): Promise<void> {
   await createModel<IRuleModel>(
     'moderation_rules',
-    rule.set('action', rule.action.toLowerCase() === 'approve' ? 'Accept' : rule.action) as any,
+    rule as any,
```

Comment:
Can we do a better job of typing this?

Reply:
I have no idea what the typing here is doing.  It looks completely wrong to me, but it works - this is the pattern used throughout this file. I'll have to come back to this one:-(

Reply:
ok, lets just add a TODO to fix/think more about the typing then?

Reply:
Added a suitably grumpy TODO....